src/geckomat/limit_proteins/flexibilizeProtConcs.m

```diff
@@ -57,7 +57,7 @@
 predGrowth = abs(sol.f);
 
 % Get those proteins with a concentration defined
-protConcs = ~isnan(model.ec.concs);
+protConcs = find(~isnan(model.ec.concs));
```

Comment:
I think that using a logical array is faster than using find after get it. But here I don't think is extreme the time increased

Reply:
It has to do with the change in line 90, which requires it to be indices not logical.